src/View/addons/ImageFusionAddOnOption.py

```diff
@@ -67,13 +72,19 @@ def setupUi(self):
         self.gridLayout.setContentsMargins(0, 0, 0, 0)
         self.gridLayout.setVerticalSpacing(0)
 
+        hspacer = None
         # Create horizontal spacer in the middle of the grid
         try:
             # PySide6 worked on Windows
-            hspacer = QtWidgets.QSpacerItem(QtWidgets.QSizePolicy.Expanding,QtWidgets.QSizePolicy.Minimum)
+            hspacer = QtWidgets.QSpacerItem(5,
```

Comment:
not clear what are appropriate preferred width and height values.

Reply:
Agreed